src/core/types/workspace.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface WorkspaceAttribute {
+  id: string;
+  name: string;
+  description?: string;
+  features?: string[];
+  color?: string;
+  icon?: string;
+  defaultVISTheme?: string;
```

Comment:
Had a discussion with team, the original requirement was to support different theme on workspace level. But we're not going to support this in our initial release. I will remove `defaultVISTheme` for now to avoid confusion.